Source/Client.swift

```diff
@@ -49,29 +49,28 @@ extension Client {
         request.host = "\(host):\(port)"
         request.userAgent = "Zewo"
 
-        if request.connection == nil {
+        if request.connection == [] {
             request.connection = "close"
         }
 
         return request
     }
 
-    public func respond(request: Request) throws -> Response {
+    public func respond(to request: Request) throws -> Response {
         let request = addHeaders(request)
-        let stream = try client.connect()
+        try self.connection.open(timingOut: .never)
```

Comment:
not sure if we should connect here every time. But it's the same behaviour as before.